solutions/devsprint-michael-douglas-2/DeliveryApp/Screens/Components/Category/CategoryCellView.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  CategoryCellView.swift
+//  DeliveryApp
+//
+//  Created by Murillo R. Araújo on 21/02/22.
+//
+
+import UIKit
+
+final class CategoryCellView: UIView {
+
+    let imageView: UIImageView = {
```

Comment:
Para garantir uma maior performance/otimização do seu código, considere utilizar `lazy var` para suas propriedades de componentes visuais.

Este artigo explica bem quando utilizar lazy var: https://www.avanderlee.com/swift/lazy-var-property/

Além das `lazy vars` otimizarem a inicialização ocorrendo somente quando a propriedade for utilizada pela primeira vez, ela ainda garante que o valor inicial não tenha que ser recalculado toda vez que você chama a sua propriedade, garantindo assim uma maior performance e experiência.

Reply:
Boa, corrigido